Personnummer/Luhn.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Linq;
+
+namespace Personnummer
```

Comment:
Extension methods are great, but for this case I'd prefer it to either just be a normal static class or for it to be kept in the `Personnummer` class.  
I feel that bloating the `string` and `int[]` types with the LuhnCheck method feels a bit much.